src/serde/mod.rs

```diff
@@ -340,4 +341,19 @@ mod tests {
             Element::read_first(to_string(&expected).unwrap())
         );
     }
+
+    #[test]
+    fn test_symbol() {
+        let i = r#"inches"#;
+        let expected = String::from("inches");
+        assert_eq!(expected, from_ion::<String, _>(i).unwrap());
```

Comment:
Forgot to add, not sure why but with `assert_eq!`, the compiler can't seem to infer `String` as the type: "cannot infer type of the type parameter `T` declared on the function `from_ion`".